typescript-data-model/order-history.ts

```diff
@@ -0,0 +1,68 @@
+interface Item {
+  itemRef: string;
+  returnable: boolean;
+  delivered: boolean;
+  cost: number;
+  // metadata here per order.
+}
+
+interface Order {
+  orderID: string;
+  orderDetails: string;
+  orderInvoice: string;
+  orderUser: string;
+  totalCharge: number;
+  orderDate: string;
+  orderItems: Item[];
+  // returnable: boolean;
+  // delivered: boolean;
+  // does the interface include the meta information such as buttons, embedded, images, and so forth?
```

Comment:
All information that will be inside the objects, but just like your other comment, that would be better suited inside the `Item` interface if it's metadata related to the specific items